docker/gunicorn.conf.py

```diff
@@ -1,11 +1,26 @@
+import base64
+import hashlib
 import multiprocessing
 import os
 from pathlib import Path
 
-VENV_BASE_DIR = Path("/simplye_venv")
-VENV_ACTUAL = [Path(VENV_BASE_DIR / d) for d
-               in os.listdir(VENV_BASE_DIR)
-               if d.startswith("simplye_app-") and os.path.isdir(VENV_BASE_DIR / d)][0]
+
+def _venv_name(app_home, file='Pipfile'):
+    """Compute the virtual environment name, given an app's directory."""
+    app_home = os.path.normpath(app_home)
+    filepath = os.path.join(app_home, file)
+    hash = base64.urlsafe_b64encode(hashlib.sha256(filepath.encode()).digest()[:6]).decode()[:8]
+    last_path_component = os.path.split(app_home)[-1]
+    return f'{last_path_component}-{hash}'
+
+# VENV_BASE_DIR = Path("/simplye_venv")
+# VENV_ACTUAL = [Path(VENV_BASE_DIR / d) for d
+#                in os.listdir(VENV_BASE_DIR)
+#                if d.startswith("simplye_app-") and os.path.isdir(VENV_BASE_DIR / d)][0]
+
+VENV_BASE = os.environ.get('WORKON_HOME', '/venv')
+APP_HOME = os.environ.get('LIBRARY_REGISTRY_HOME', '/apps/library-registry')
+APP_VENV = os.path.join(VENV_BASE, os.environ.get('LIBRARY_REGISTRY_VENV', _venv_name(APP_HOME)))
```

Comment:
Should we add documentation for these new environment variables?

Reply:
Sure. I think I'll rename a couple of these to make it more clear that they are docker-specific.